### PR TITLE
Added status information to both hooks

### PR DESCRIPTION
**Pull request by royanger**

### CODE REVIEW

**Comment by royanger** [comment]
@BRKalow, I see that `abandoned` is a one of the statuses for useSignUp from https://github.com/clerkinc/javascript/blob/7f4d4b942e8834462cdc0976b106d9739c345f6b/packages/types/src/signUp.ts#L104, but I don't see how it is set. The only instances of 'abandoned' in the JS repo are types. Is this outdated, or set another way?

**Review by rachelnabors — APPROVED** [review]

File: docs/references/react/use-sign-in.mdx [hunk 1]
```diff
@@ -175,3 +175,12 @@ The `useSignIn()` hook provides access to the [`SignIn`](/docs/references/javasc
...
 | `setSession()` | A function that sets the active session. |
 | `signIn` | An object that contains the current sign-in attempt status and methods to create a new sign-in attempt. |
+
+### Result Status
+
+| Values | Descriptiom |
+| --- | --- |
+| `complete` | User has been signed in and custom flow can proceed to `setActive()` to create session. |
+| `needs_factor_one` | The First Factor verification is missing. One of `email_link`, `email_code`, `phone_code`, `web3_metamask_signature` or `oauth_provider` is required to verify user. See [First Factor](/docs/references/javascript/sign-in/first-factor) for details. |
+| `needs_factor_two` | The Second Factor verification is missing. The Second Factor is a optional step to provide additional verification and includes `phone_code` and `totp`. See [Second Factor](/docs/references/javascript/sign-in/second-factor) for details.|
```
Comment (alexisintech) [inline docs/references/react/use-sign-in.mdx:185]:
```suggestion
| `needs_factor_two` | The Second Factor verification is missing. The Second Factor is an optional step to provide additional verification and includes `phone_code` and `totp`. See [Second Factor](/docs/references/javascript/sign-in/second-factor) for details.|
```

File: docs/references/react/use-sign-in.mdx [hunk 2]
```diff
@@ -175,3 +175,12 @@ The `useSignIn()` hook provides access to the [`SignIn`](/docs/references/javasc
 | `setActive()` | A function that sets the active session. In most cases, this should be used over `setSession()`. |
 | `setSession()` | A function that sets the active session. |
 | `signIn` | An object that contains the current sign-in attempt status and methods to create a new sign-in attempt. |
+
+### Result Status
+
+| Values | Descriptiom |
+| --- | --- |
+| `complete` | User has been signed in and custom flow can proceed to `setActive()` to create session. |
```
Comment (alexisintech) [inline docs/references/react/use-sign-in.mdx:183]:
```suggestion
| `complete` | The user has been signed in and custom flow can proceed to `setActive()` to create session. |
```

File: docs/references/react/use-sign-up.mdx [hunk 3]
```diff
@@ -181,3 +181,11 @@ The `useSignUp()` hook gives you access to the [`SignUp`](/docs/references/javas
 | `setSession()` | A function that sets the active session. |
 | `signUp` | An object that contains the current sign-up attempt status and methods to create a new sign-up attempt. |
 
+### Result Status
+
+| Values | Descriptiom |
+| --- | --- |
+| `complete` | User has been created and custom flow can proceed to `setActive()` to create session. |
```
Comment (alexisintech) [inline docs/references/react/use-sign-up.mdx:188]:
```suggestion
| `complete` | The user has been created and custom flow can proceed to `setActive()` to create session. |
```

File: docs/references/react/use-sign-up.mdx [hunk 4]
```diff
@@ -181,3 +181,11 @@ The `useSignUp()` hook gives you access to the [`SignUp`](/docs/references/javas
...
+| `abandoned` | The SignUp attempt will be abandoned if it was started more than 24 hours previously. |
```
Comment (alexisintech) [inline docs/references/react/use-sign-up.mdx:189]:
```suggestion
| `abandoned` | The sign-up attempt will be abandoned if it was started more than 24 hours previously. |
```

File: docs/references/react/use-sign-up.mdx [hunk 5]
```diff
@@ -181,3 +181,11 @@ The `useSignUp()` hook gives you access to the [`SignUp`](/docs/references/javas
...
+| `missing_requirements` | A requirement from the [Email, Phone, Username](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username) settings is missing. EG, in your Dashboard the Password setting is required but a password as not provided in the custom flow. |
```
Comment (alexisintech) [inline docs/references/react/use-sign-up.mdx:190]:
```suggestion
| `missing_requirements` | A requirement from the [Email, Phone, Username](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username) settings is missing. For example, in the Clerk Dashboard, the Password setting is required but a password was not provided in the custom flow. |
```

**Review by alexisintech — CHANGES_REQUESTED** [review]

**Comment by BRKalow** [comment]
@royanger This value comes from FAPI I believe, so we likely would not set it explicitly anywhere in our SDKs.

**Comment by royanger** [comment]
> @royanger This value comes from FAPI I believe, so we likely would not set it explicitly anywhere in our SDKs.

I actually got Pantelis and Giannis to help out. Thanks for replying. https://clerkinc.slack.com/archives/C05LHV6AK50/p1698685499922449 I didn't add it to SignIn since its not in the type def.

**Review by alexisintech — APPROVED** [review]
😸💖

**Pull request merged**